contracts/Synthetix.sol

```diff
@@ -10,6 +10,7 @@ import "./interfaces/ISupplySchedule.sol";
 
 
 // https://docs.synthetix.io/contracts/source/contracts/synthetix
+// TODO: apparently this is already too big of a contract?
```

Comment:
I'm pretty worried both Synthetix and Exchanger will get too big, but let's see.